hs_core/views/utils.py

```diff
@@ -1,11 +1,17 @@
 from __future__ import absolute_import
 
+from django.contrib.auth.models import Group, User
 from django.core.exceptions import PermissionDenied
 from django.core import exceptions
 from hs_core import hydroshare
 from ga_resources.utils import get_user
 import json
-
+from hs_core.models import ResourceFile
+from hs_core.signals import pre_add_files_to_resource, post_add_files_to_resource
+from django.core.files import File
+from django.core.files.uploadedfile import UploadedFile
+from rest_framework.views import exception_handler
+from rest_framework.exceptions import *
```

Comment:
The django imports should be with the imports of django above, hte rest_framework imports should be after that, adn the hs_core imports should be last.